monai/inferers/utils.py

```
@@ -38,6 +38,72 @@
 
 __all__ = ["sliding_window_inference"]
 
+def ensure_channel_first(
+    x: torch.Tensor,
+    spatial_ndim: Optional[int] = None,
```

Ruff (0.12.2)

Undefined name `Optional`

(F821)